openedx_tagging/core/tagging/import_export/api.py

```diff
@@ -48,12 +48,32 @@
 
 from django.utils.translation import gettext as _
 
+from .. import api as taxonomy_api
 from ..models import TagImportTask, TagImportTaskState, Taxonomy
 from .exceptions import TagImportError
 from .import_plan import TagImportPlan, TagImportTask
 from .parsers import ParserFormat, get_parser
 
 
+def create_taxonomy_and_import_tags(
+    taxonomy_name: str,
+    taxonomy_description: str,
+    file: BytesIO,
+    parser_format: ParserFormat
+) -> bool:
+    """
+    Create a taxonomy and import the tags from `file`
+    """
+    taxonomy = taxonomy_api.create_taxonomy(taxonomy_name, taxonomy_description)
+
+    import_success = import_tags(taxonomy, file, parser_format)
+
+    if not import_success:
+        taxonomy.delete()
+
+    return import_success
```

Comment:
The [issue](https://github.com/openedx/modular-learning/issues/115) says:

> 7. If an error occurs, display an alert/toast and put the details in the browser console.

So we need to send any errors logged during import back to the frontend so it can `console.error` it.

I believe that means changing `import_tags` so that it returns the task or the error log as well as the status.